### PR TITLE
Get image tag from deploy_app task inputs

### PR DESCRIPTION
**Pull request by petechd**
### What is the context of this PR?
This is complementary PR for the eq-pipelines changes in [this PR](https://github.com/ONSdigital/eq-pipelines/pull/247).

### How to review 
Fly your pipeline using image of the app in this PR or check [my pipeline where I used these settings](https://concourse.gcp.onsdigital.uk/teams/eq/pipelines/pete-test-env-on-release).

### Checklist

* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

**Comment by petechd** [comment]
[Create eq-test environment (2 SP)](https://trello.com/c/ZgTGzuZq/4787-create-eq-test-environment-2-sp)

File: ci/deploy_app.yaml [hunk 1]
```diff
@@ -6,6 +6,7 @@ image_resource:
     tag: slim
 inputs:
   - name: eq-questionnaire-runner
+  - name: image_tag
```
Comment (MebinAbraham) [inline ci/deploy_app.yaml:9]: As per https://github.com/ONSdigital/eq-pipelines/pull/252#pullrequestreview-720278405, you should make this input optional and fetch from file if it exists, if not fetch from env vars. If neither exist, then it should fail the job.

Comment (petechd) [reply ci/deploy_app.yaml]: Made optional and tested both scenarios [here for test env](https://concourse.gcp.onsdigital.uk/teams/eq/pipelines/pete-test-env-on-release/jobs/deploy-questionnaire-runner/builds/98) and [here for staging](https://concourse.gcp.onsdigital.uk/teams/eq/pipelines/pete-test-env-on-release/jobs/deploy-questionnaire-runner/builds/97). Thanks!

**Review by MebinAbraham — COMMENTED** [review]
The CI README should be updated to mention that when fly executing the image tag can be passed in as in env var or as in file input located at `./image-tag/tag`

**Comment by petechd** [comment]
> or as in file input located at `./image-tag/tag`

Added text to `Deploying the app` section of that README.

**Review by MebinAbraham — APPROVED** [review]

**Review by pricem14pc — APPROVED** [review]
Tested successfully in an eq-test (image-tag file) and eq-staging (default to `latest`) pipeline. The expected image is deployed correctly.

**Pull request merged**